### DISCUSSION TITLE
After Update to 0.17.1 and waiting 1 Day, no mesh client connects anymore.

### DISCUSSION THREAD

**Discussion by bloesss** [amidaware/tacticalrmm#1687] [Q&A] [answered]
Hi,

I updated the server from 0.15.0 to 0.17.1, because the SSL Certificate would end up the next day and I used the maintenance to update the server.

Everythin went fine, even the agents updated theirselves to 2.5.0.

But at the moment, the old certificate would end (I renewed it the day before and it worked!), no Clients connect anymore.
The Agent log just says "Service started". I don't find any error logs on server side.

If I install a new client, the new client works. Until I restart the Server. At this point, the client won't connect anymore and the Client Version (Windows) is shown as "0.1.0" at server side.

Where can I learn more to bring it back to life? Are there any log files that could help?

**Comment by dinger1986** [comment]
How did you renew your certificate?sounds like mesh maybe isn't picking it up?

**Reply by bloesss** [reply]
I do not use the integrated Meshcentral and installed all agents with --nomesh.

I followed those instructions, including the certificate Renewal:

https://docs.tacticalrmm.com/update_server/

**Reply by dinger1986** [reply]
Did you follow all the instructions including update with force?

**Reply by dinger1986** [reply]
Also if mesh isn't installed how would it work?

**Reply by bloesss** [reply]
I did Not use the Force Parameter as the Update went through without.

After the Update, everything was Fine until the exact Date and time the old, no more used certificate Ended.

we Are using our own meshcentral Installation and don’t Need the one with trmm.

**Reply by bloesss** [reply, marked as answer]
After running the update skript again, everything works fine. Thanks for the support.
